Misc/NEWS.d/next/Library/2021-12-14-13-15-41.bpo-45995.Am9pNL.rst

```diff
@@ -0,0 +1,3 @@
+Add a "z" option to the string formatting specification that coerces negative
+zero floating-point values to positive zero after rounding to the format
+precision.
```

Comment:
@belm0: You might want to give yourself credit here. "Contributed by Joen Belmonte in bpo-45995.", or similar.